### PR TITLE
IAM Custom Role - documentation edited (note ???'d question)

### PR DESCRIPTION
**Pull request by alexander-levitski**

### CODE REVIEW

File: community/cloud-foundation/templates/iam_custom_role/organization_custom_role.py.schema [hunk 1]
```diff
@@ -13,26 +13,26 @@
...
 info:
-  title: IAM Organization Level Custom Roles
+  title: Custom IAM Role (Organization Level)
   author: Sourced Group Inc.
   description: |
-    Creates a custom organizational role under the specified organization ID.
+    Creates a custom organization-level IAM role under the specified organization ID.
 
-    For more information on this resource:
+    For more information on this resource: THE LINK IS TO .../projects... SHOULD BE .../organizations...???
```
Comment (sourced-vince) [inline community/cloud-foundation/templates/iam_custom_role/organization_custom_role.py.schema]: I'm not sure I follow your question here. Could you please clarify.

File: community/cloud-foundation/templates/iam_custom_role/organization_custom_role.py.schema [hunk 2]
```diff
@@ -13,26 +13,26 @@
...
     https://cloud.google.com/iam/reference/rest/v1/projects.roles/create
 
-    To deploy custom organizational roles your project will need to enable the
-    Identity and Access Management (IAM) API. You may also need to grant
+    To deploy custom organization-level roles, your project OR ORGANIZATION??? needs the Identity
```
Comment (sourced-vince) [inline community/cloud-foundation/templates/iam_custom_role/organization_custom_role.py.schema]: `your project needs the Identity...`

File: community/cloud-foundation/templates/iam_custom_role/project_custom_role.py.schema [hunk 3]
```diff
@@ -13,26 +13,26 @@
...
   author: Sourced Group Inc.
   description: |
-    Creates a custom project role under the specified organization ID.
+    Creates a custom project-level IAM role under the specified organization ID.
 
     For more information on this resource:
     https://cloud.google.com/iam/reference/rest/v1/projects.roles/create
 
-    To deploy custom project roles your organization will require the Identity
+    To deploy custom project-level roles, your organization ??? OR PROJECT??? needs the Identity
```
Comment (sourced-vince) [inline community/cloud-foundation/templates/iam_custom_role/project_custom_role.py.schema]: `your project needs the Identity...`

File: community/cloud-foundation/templates/iam_custom_role/README.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-Templated custom IAM role deployment
+This template creates a custom IAM role for an organization or a project.
 
 ## Prerequisites
 
 - Install [gcloud](https://cloud.google.com/sdk)
-- Create a [GCP project, setup billing, enable requisite APIs](../project/README.md)
-
+- Create a [GCP project, set up billing, enable requisite APIs](../project/README.md)
+- ??? roles/permissions ???
```
Comment (sourced-vince) [inline community/cloud-foundation/templates/iam_custom_role/README.md]: - Grant the [iam.roleAdmin, iam.organizationRoleAdmin or owner ](https://cloud.google.com/iam/docs/understanding-custom-roles) IAM role to the project service account

File: community/cloud-foundation/templates/iam_custom_role/organization_custom_role.py.schema (hunk 1 above)
Comment (sourced-vince) [reply community/cloud-foundation/templates/iam_custom_role/organization_custom_role.py.schema]: use this link instead:

https://cloud.google.com/iam/docs/understanding-custom-roles

**Comment by alexander-levitski** [comment]
Done. Approve?

**Pull request closed without merging**